projects/PackagesWithAnalyzers/PackagesWithAnalyzers.csproj

```diff
@@ -37,7 +37,7 @@
     <PackageReference Include="Ardalis.ApiEndpoints.CodeAnalyzers" Version="*" PrivateAssets="All" IncludeAssets="runtime; build; native; contentfiles; analyzers; buildtransitive" />
     <PackageReference Include="FakeItEasy.Analyzer.CSharp" Version="*" PrivateAssets="All" IncludeAssets="runtime; build; native; contentfiles; analyzers; buildtransitive" />
     <PackageReference Include="Lucene.Net.Analysis.Common" Version="4.8.0-beta00016" PrivateAssets="all" IncludeAssets="runtime; build; native; contentfiles; analyzers; buildtransitive" />
-    <PackageReference Include="Libplanet.Analyzers" Version="*" PrivateAssets="All" IncludeAssets="runtime; build; native; contentfiles; analyzers; buildtransitive" />
+    <PackageReference Include="Libplanet.Analyzers" Version="2.5.0" PrivateAssets="All" IncludeAssets="runtime; build; native; contentfiles; analyzers; buildtransitive" />
```

Comment:
Why is this fixed?

Reply:
`Libplanet.Analyzers` has a dependency on `Libplanet`, meaning if `*` pulls in a newer version than the `Libplanet` version, the build task will start complaining that the version of `Libplanet` was downgraded.